cmd/carbonzipper/main.go

```diff
@@ -67,7 +68,16 @@ func main() {
 			zap.Error(err),
 		)
 	}
+	expvar.NewString("BuildVersion").Set(BuildVersion)
+	logger.Info("starting carbonzipper",
+		zap.String("build_version", BuildVersion),
+		zap.Any("zipperConfig", config),
+	)
 
-	zipper.StartCarbonZipper(config, logger)
+	app, err := zipper.InitializeApp(config, logger, BuildVersion)
+	if err != nil {
+		logger.Error("Error initializing app")
+	}
+	app.StartCarbonZipper()
```

Comment:
nit: any reason for `Start` for carbonapi vs `StartCarbonZipper` here?

Reply:
no, there isn't. Will update them to be the same.